src/main/java/com/rakovets/course/java/core/practice/jcf_collection/NotValidWordLengthException.java

```diff
@@ -0,0 +1,7 @@
+package com.rakovets.course.java.core.practice.jcf_collection;
```

Comment:
"Violation: Contains code not related to this task
Solution: Remove code not related to this task"